### ISSUE TITLE
missing alt attribute for MyWay link

### ISSUE THREAD

**Issue by suzannezc** [wrdsb/wordpress-theme#384]

**Comment by suzannezc** [comment]
In widget, not in home.php, but should review the alt attributes for all buttons for meaning
